mzLib/FlashLFQ/IsoTracker/SearchingTarget.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace FlashLFQ.IsoTracker
+{
+    public class SearchingTarget
+    {
+        public List<char> TargetMotifs; // motif string list
+
+        /// <summary>
+        /// Directly use the motif string to create a SearchingTarget
+        /// </summary>
+        /// <param name="motifs"></param>
+        /// <param name="option2"></param>
+        public SearchingTarget(List<char> motifs)
+        {
+            TargetMotifs = motifs;
+        }
+
+        /// <summary>
+        /// Filter the peptide sequence based on the modification motifs
+        /// </summary>
+        /// <param name="peptideSequence"></param>
+        /// <returns></returns>
+        public bool MotifFilter(string peptideSequence)
```

Comment:
Can we change this to `ContainsAcceptableModifiedResidue` or something like that to be more informative.